attractions/snackbar/snackbar.svelte

```diff
@@ -6,14 +6,47 @@
 
   let _class = null;
   export { _class as class };
+  /**
+   * A class string to add to the label of the snackbar.
+   * @type {string}
+   */
   export let textClass = null;
+  /**
+   * A class string to add to the action button of the snackbar.
+   * @type {string}
+   */
   export let buttonClass = null;
 
+  /**
+   * The text to show on the snackbar.
+   * @type {string}
+   */
   export let text;
+  /**
+   * The action for the button on the snackbar. If this is null, the button is not rendered. Otherwise it has text as a label and calls callback on click.
+   * @type {{text: string; callback: () => void}}
+   */
   export let action = null;
+  /**
+   * Whether to call the `closeCallback` when an action button is pressed, after its callback.
+   * @type {boolean}
+   */
   export let closeOnAction = true;
+  /**
+   * The callback to call to close the snackbar in the `SnackbarContainer`.
+   * Passed internally by the `SnackbarContainer` when using the `showSnackbar` function.
+   * @type {() => void}
+   */
   export let closeCallback;
+  /**
+   * The transition to use to animate the in and out of the snackbar.
+   * @type {() => import('svelte/transition').TransitionConfig}
```

Comment:
This typing will forbid passing parameters to the transition, we should use this instead:
```suggestion
   * @type {(...args: any[]) => import('svelte/transition').TransitionConfig}
```